src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/BalanceCalculatorDeleteActionIntegrationTest.java

```diff
@@ -46,13 +46,19 @@ void calculate_deleteTimeEntryOneDay_contributionsAndCumulativeTotalsAsExpected(
     assertTotals(accruals.get(1), 240, 7200);
     assertTotals(accruals.get(2), 720, 7920);
     assertTotals(accruals.get(3), 120, 8040);
+
+    // Night Hours
+    assertTotals(accruals.get(4), 0, 1000);
```

Comment:
I'll apologies immediately as I don't have an alternative to propose but I'm conscious that we'll be adding up to 8 more accrual types, is this the correct test structure to use/start with?
Should each accrual have it's own test to keep the tests simple?

Reply:
We do have unit tests for each accrual, but for integration tests we decided that having a time entry cover multiple accrual types is the most realistic scenario

Reply:
As we add more accrual types I don't think it will be possible to have a single integration test cover every type, so I imagine then we'd need more tests with their own mappings

Reply:
Was also thinking that if we ever change functionality later i.e. the boundaries for when night hours are accrued, it would be cleaner to see that only night hour tests changed i.e. if each accrual had their own tests it would be really easy to see that the code we changed for night hours didn't impact any other accruals.

Reply:
Do you mean to keep the test set up the same but have the asserts in their own tests?

Reply:
Moved them into their own tests now